### PR TITLE
[LUM-272] Wallet -> Show a popup at refresh

### PR DESCRIPTION
**Pull request by greedyboi**

### CODE REVIEW

File: src/navigation/Layout/MainLayout/MainLayout.tsx [hunk 1]
```diff
@@ -29,6 +30,26 @@ class MainLayout extends PureComponent<Props> {
...
+
+                        return (e.returnValue = '');
+                    };
+                }
+            } else {
+                window.onbeforeunload = null;
+            }
+        }
+    }
+
```
Comment (ThibaultJRD) [inline src/navigation/Layout/MainLayout/MainLayout.tsx:52]: Why this component is a class and not a function?

**Review by ThibaultJRD — CHANGES_REQUESTED** [review]
Update all `Class` components to `Function` components

File: src/navigation/Layout/MainLayout/MainLayout.tsx (hunk 1 above)
Comment (greedyboi) [reply src/navigation/Layout/MainLayout/MainLayout.tsx]: Old implementation, fixed 😊

**Review by ThibaultJRD — APPROVED** [review]
🚀

**Pull request merged**